aws/cloudformation/standalone/marketing/deployment/shared-infra.yml.erb

```diff
@@ -0,0 +1,33 @@
+# Marketing Root DNS Resources
+# This template is manually deployed to create/update the Marketing App's AWS resources.
```

Comment:
Per our discussion, this stack template can go away.

The hosted zone for marketingsite.code.org (and marketingsite.<env>-code.org) will be created in a separate Infra stack, and can be referenced in the `marketing.yml.erb` to create the A record.

Reply:
I repurposed the shared infra stack for the TLS Certs, though I can change it to a cert specific stack if you'd like